pkg/mc/mc.go

```diff
@@ -112,7 +113,8 @@ mc -r kind -o json -- get pods -n kube-system | jq 'keys[] as $k | "\($k) \(.[$k
 	}
 
 	cmd.Flags().StringVarP(&mc.Regex, "regex", "r", mc.Regex, "a regex to filter the list of context names in kubeconfig. If not given all contexts are used")
-	cmd.Flags().StringVarP(&mc.NegRegex, "negative-regex", "n", mc.NegRegex, "a regex to exclude matches from the result set. Evaluated succeeding to the including regex filter")
+	cmd.Flags().StringVarP(&mc.NegRegex, "negative-regex", "x", mc.NegRegex, "a regex to exclude matches from the result set. Evaluated succeeding to the including regex filter")
+	cmd.Flags().StringVarP(&mc.Namespaces, "namespaces", "n", mc.Namespaces, "(optional) comma-separated list of namespaces. Overrides namespace(s) specified in kubectl command")
```

Comment:
As much as I hate breaking the API here, I actually do agree that calling this `n` makes more sense.